Platform/ElkhartlakeBoardPkg/BoardConfig.py

```diff
@@ -210,6 +210,10 @@ def GetPlatformDsc (self):
         if self.BUILD_CSME_UPDATE_DRIVER:
             common_libs.append ('MeFwUpdateLib|Silicon/$(SILICON_PKG_NAME)/Library/MeFwUpdateLib/MeFwUpdateLib.inf')
         dsc['LibraryClasses.%s' % self.BUILD_ARCH] = common_libs
+        if self.RELEASE_MODE == True:
+            dsc['PcdsFixedAtBuild'] = [
+                'gPlatformElkhartLakeTokenSpaceGuid.PcdIsReleaseBuild | TRUE'

```

Comment:
Why do we introduce a new PCD for DEBUG / RELEASE build ?
There is existing way to do that using "DEBUG_CODE".

Reply:
Ok noted will use DebugCodeEnabled() instead